src/Utils/JsonUtils.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LupaSearch\Utils;
+
+use InvalidArgumentException;
+
+use function json_decode;
+use function json_encode;
+use function json_last_error;
+use function json_last_error_msg;
+
+use const JSON_ERROR_NONE;
+
+class JsonUtils
+{
+    /**
+     * @throws InvalidArgumentException
+     */
+    public static function jsonDecode(string $json, bool $assoc = false, int $depth = 512, int $options = 0)
```

Comment:
if you don't use depth, assoc and options - then why do you include it here?

Reply:
https://github.com/lupasearch/lupasearch-php-client/pull/4/files/fba67d06b055d6f4c49c49df58454a46c98fd789#diff-13e56c1807e8990294eabfe8de13dd47f97c58f10855351bdc5a188acf4d4ae4R23